lte/gateway/deploy/roles/agw_docker/vars/all.yaml

```diff
@@ -21,7 +21,8 @@ magma_pkgrepo_key: "https://artifactory.magmacore.org:443/artifactory/api/gpg/ke
 magma_pkgrepo_key_fingerprint: ""
 magma_use_pkgrepo: yes
 
-docker_registry: ""
+docker_registry: "docker.artifactory.magmacore.org/"
```

Comment:
What are the implications of that change on master

Reply:
Nothing, we will only be using this value when we deploy the AGW docker script and try to pull the latest docker images. We can tag our 1.8.0 release as latest, so the master branch will use that code for deployment.

Reply:
Isn't that the better value for now, until we have anything published as latest?

Reply:
What about "agw-test.artifactory.magmacore.org"? That's where the build-all job pushes its images.

Reply:
If we want to build fresh docker images using master branch and run them locally, then keeping it `""` is a good option.

Reply:
@sebathomas then we will also update the tag.

Reply:
but these images are not tagged as `latest`? Maybe we should change that. But that would still pull an untagged and uncontrolled state. I would leave it empty for now and think about that later

Reply:
but are they also getting the right `docker_registry:` value? If not, it will not work. That is the part I do not understand and where I not see the benefit of changing that line here